lib/manageiq/release/pull_request_blaster_outer.rb

```diff
@@ -3,10 +3,23 @@
 module ManageIQ
   module Release
     class PullRequestBlasterOuter
-      attr_reader :repo, :base, :head, :script, :dry_run, :message
+      attr_reader :repo, :base, :head, :script, :dry_run, :message, :assignee, :labels, :pr_body
 
       ROOT_DIR = Pathname.new(__dir__).join("..", "..", "..").freeze
-      def initialize(repo, base:, head:, script:, dry_run:, message:)
+
+      def self.run(opts)
+        results = {}
+
+        ManageIQ::Release.each_repo(opts[:repo]) do |repo|
```

Comment:
Pattern wise, this bit has lived in the bin files.  This is intentional as we may have to run multiple units on a particular repo